MonoGame.Framework.Content.Pipeline/Builder/ContentBuilder.cs

```diff
@@ -0,0 +1,390 @@
+// MonoGame - Copyright (C) MonoGame Foundation, Inc
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+using System.Reflection;
+using Microsoft.Xna.Framework.Content.Pipeline;
+using Microsoft.Xna.Framework.Content.Pipeline.Serialization.Compiler;
+using MonoGame.Framework.Content.Pipeline.Builder.Server;
+
+namespace MonoGame.Framework.Content.Pipeline.Builder;
```

Comment:
`MonoGame.Framework.Content.Pipeline.Builder.ContentBuilder` sounds weird.

Should this maybe move up to `MonoGame.Framework.Content.Pipeline` and leave `MonoGame.Framework.Content.Pipeline.Builder` for the old stuff?

Reply:
This definitely should not be top level as no importer / processor would try to access this stuff.